recipes/lapack95/make.inc

```diff
@@ -0,0 +1,46 @@
+#
+#  -- LAPACK95 interface driver routine (version 2.0) --
+#     UNI-C, Denmark; Univ. of Tennessee, USA; NAG Ltd., UK
+#     August 5, 2000
+#
+FC_fr	 = $(FC) -ffree-form
+FC_ff    = $(FC) -ffixed-form
+# -dcfuns  Enable recognition of non-standard double
+#          precision  complex intrinsic functions
+# -dusty   Allows the compilation and execution of "legacy"
+#          software by downgrading the category  of  common
+#          errors  found  in  such software from "Error" to
+# -ieee=full enables all IEEE arithmetic facilities
+#          including non-stop arithmetic.
+
+OPTS0    = -pipe
+MODLIB   = -I$(PREFIX)/include
+OPTS1    = -c -O3 $(OPTS0)
+OPTS3    = $(OPTS1) $(MODLIB)
+OPTL     = -o 
+OPTLIB   =
+
+LAPACK_PATH = $(PREFIX)/lib
+
+LAPACK95 = ../lapack95.a
+LAPACK77 = $(LAPACK_PATH)/liblapack.so
```

Comment:
```suggestion
LAPACK77 = $(LAPACK_PATH)/liblapack$(SHLIB_EXT)
```

Reply:
Addressed via https://github.com/conda-forge/staged-recipes/pull/14212/commits/38b23a22c79870d6858d239a949e8b6f7a6d0b3b for LAPACK and BLAS.